experimental/i18n-dev.crontab

```diff
@@ -15,21 +15,21 @@
 #
 # Where can I ask a question about this config?
 # Reach out to the #i18n slack channel.
-
+PATH=/home/ubuntu/.rbenv/shims:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 HOME=/home/ubuntu
 
 # Run a CI Build to update the server from staging. We do this primarily to
 # make sure the server gets seeded with latest level content for the sync in.
 # Note that the CI_ONLY_BUILD environment variable is necessary; the server is
 # not set up to actually serve a version of the website, and it will fail to
 # build correctly if not thus restricted.
-0 */4 * * * BASH_ENV=~/.bashrc bash -lc "cd /home/ubuntu/code-dot-org && CI_ONLY_BUILD=true bundle exec ./bin/cronjob ./aws/ci_build >>/tmp/cronlog 2>&1"
+0 */4 * * * bash -l -c "cd /home/ubuntu/code-dot-org && CI_ONLY_BUILD=true bundle exec ./bin/cronjob ./aws/ci_build >>/tmp/cronlog 2>&1"
```

Comment:
I'm actually not sure we need `bash -l -c` anymore if we are setting the PATH variable directly in this crontab file?

Reply:
I agree.. but now that it's working, I don't want to mess it up. 😂